task_4/Tatiana.Derevenets/src/main/java/task_4/Tip.java

```diff
@@ -0,0 +1,30 @@
+package task_4;
+
+public class Tip{
+
+  public static double calcTip(double bill, String level) {
+
+    double result;
+    switch (level){
+      case "terrible":
+        result = bill;
+        break;
+      case "poor":
+        result = bill * 0.05;
+        break;
+      case "good":
+        result = bill * 0.1;
+        break;
+      case "great":
+        result = bill * 0.15;
+        break;
+      case "excellent":
+        result = bill * 0.2;
+        break;
+      default:
+        result = 0;

```

Comment:
You can use:
throw new IllegalArgumentException("Wrong tips type");
instead of **result = 0**